apps/documents/rpc.py

```diff
@@ -46,3 +46,24 @@ def add_document_from_s3(self, bucket, key, vault_wallet, storage_credentials, v
         log_metric('transmission.error', tags={'method': 'event_rpcclient.subscribe', 'code': 'RPCError',
                                                'module': __name__})
         raise RPCError("Invalid response from Engine")
+
+    def put_document_in_s3(self, bucket, key, vault_wallet, storage_credentials, vault, document_name):
+        LOG.debug(f'Telling Engine to put doc: {document_name} from vault: {vault} with {key} to s3 bucket: {bucket}')
+        log_metric('transmission.info', tags={'method': 'document_rpcclient.put_document_in_s3',
+                                              'module': __name__})
+
+        result = self.call('vault.put_document_in_s3', {
+            "bucket": bucket,
+            "key": key,
+            "vaultWallet": vault_wallet,
+            "storageCredentials": storage_credentials,
+            "vault": vault,
+            "documentName": document_name,
+        })
+
+        if result.get('success', None):
+            return
+
+        log_metric('transmission.error', tags={'method': 'event_rpcclient.subscribe', 'code': 'RPCError',
```

Comment:
I just noticed that these metrics are wrong in documents rpc_client:
```suggestion
        log_metric('transmission.error', tags={'method': 'document_rpcclient.put_document_in_s3', 'code': 'RPCError',
```